google-datacatalog-hive-connector/tests/google/datacatalog_connectors/hive/prepare/datacatalog_entry_factory_test.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import unittest
+
+from google.datacatalog_connectors.hive.prepare import \
+    datacatalog_entry_factory
+
+
+class DataCatalogEntryFactoryTestCase(unittest.TestCase):
+    __MODULE_PATH = '{}'.format(os.path.dirname(os.path.abspath(__file__)))
+
+    __PROJECT_ID = 'test_project'
+    __LOCATION_ID = 'location_id'
+    __METADATA_SERVER_HOST = 'metadata_host'
+    __ENTRY_GROUP_ID = 'sql_database'
+
+    def test_make_entry_for_database_should_set_all_available_fields(
+            self):  # noqa:E125
+        databases = convert_json_to_metadata_object(
+            retrieve_json_file('databases_with_one_table.json'))['databases']
+
+        factory = datacatalog_entry_factory.DataCatalogEntryFactory(
+            DataCatalogEntryFactoryTestCase.__PROJECT_ID,
```

Comment:
```suggestion
            self.__PROJECT_ID,
```
Same question...

Reply:
Not sure if I'm viewing something outdated, but looks like this change has not been applied...

Reply:
You are right, thanks! It was only applied to `assembled_entry_factory_test.py`